File: 12-03-18/python/01-Monks-love-for-food.py
st = []
for _ in range( int(input()) ):
    t = input().split()
    if(t[0]=="1"):
        if(len(st)!=0):
            print(st.pop())
        else:
            print("No Food")
    else:
        st.append(t[1])
    

File: 12-03-18/java/01-Monks-love-for-food.java
import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        Stack<Integer> st=new Stack<Integer>();
        int q=input.nextInt();
        while(q-->0)
        {
            int t=input.nextInt();
            if(t==1)
            {
                if(!st.empty())
                    System.out.println(st.pop());
                else
                    System.out.println("No Food");
            }
            else
            {
                int c=input.nextInt();
                st.push(c);
            }
            
        }
        
    }
}
